src/bci_build/package/appcontainers.py

```diff
@@ -41,7 +41,7 @@ def _envsubst_pkg_name(os_version: OsVersion) -> str:
 PCP_CONTAINERS = [
     ApplicationStackContainer(
         name="pcp",
-        pretty_name="Performance Co-Pilot (pcp)",
+        pretty_name="Performance Co-Pilot",
```

Comment:
> WIP: dummy change

is this commit supposed to be here?

Reply:
Nope, I added that to get an actual change. Let me drop it.